test/absorption_comparison_funcs.jl

```diff
@@ -106,7 +107,22 @@ function _semi_realisitic_dens(ne::F, fion::F= 0.02, HydrogenMassFrac::F= 0.76)
     (nH_I, nH, ρ)
 end
 
-# Now actually define the functions that compute the opacities in the form comparable with Gray05
+# Gray05 uses some confusing terminology and notation.
+# - he defines the "atomic absorption coefficient" as the area per absorber and represent it with
+#   the variable α. For reference, Rybicki & Lightman call this same quantity the cross-section and
+#   represent it with the variable σ_ν.
+# - he defines the "continuous absorption coefficient per neutral hydrogen atom" and denotes it
+#   with κ. Rybicki & Lightman don't explicitly define this quantity, but it's compatible with
+#   their nomenclature. R&L would express this quantity as α_ν/nH_I (in their notation)
+# - he uses κ_ν to denote the mass opacity (just like Rybicki & Lightman)
+#
+# Now we actually define the functions that compute the opacities in the form comparable with
+# Gray05. These functons return the "continuous absorption coefficient per neutral hydrogen atom"
+# divided by Pₑ, the partial electron pressure.
+#
+# For the sake of clarity, we label Rybicki & Lightman's α_ν = κ_ν*ρ as the "linear absorption
+# coefficient" within these functions (note: throughout the rest of the codebase, we interchangably
+# use the simpler term, "absorption coefficient", to refer to this same quantity)
```

Comment:
How would you feel if I lifted this and placed it in the dev docs at a later point?

Reply:
(and removed it/referenced the dev docs here)